stubs/config/queue.stub

```diff
@@ -1,11 +1,18 @@
-import Env from '@ioc:Adonis/Core/Env'
-import type { QueueConfig } from '@ioc:Rlanz/Queue'
+{{{
+  exports({ to: app.configPath('queue.ts') })
+}}}
+import env from '#start/env'
+import { defineConfig } from '@rlanz/bull-queue'
 
-const queueConfig: QueueConfig = {
+declare module 'Rlanz/Queue' {
+  interface JobsList {}
+}
+
+const queueConfig = defineConfig({
   connection: {
-    host: Env.get('QUEUE_REDIS_HOST'),
-    port: Env.get('QUEUE_REDIS_PORT'),
-    password: Env.get('QUEUE_REDIS_PASSWORD'),
+    host: env.get('QUEUE_REDIS_HOST'),
+    port: Number.parseInt(env.get('QUEUE_REDIS_PORT') || '6379', 10),
+    password: env.get('QUEUE_REDIS_PASSWORD'),
   },
 
   queue: {},
```

Comment:
this, along with `worker: {}`, is currently showing a Typescript error, since it's expected to be of type `QueueOptions`. Not sure how this should be treated